### PR TITLE
hw6 Routing

### PR DESCRIPTION
**Pull request by vm137**

### CODE REVIEW

**Comment by vm137** [comment]
I try to test Router like this but it doesn't work for some reason. 
```
import SearchBlock from '../SearchBlock/SearchBlock';

  test('render with <SearchBlock /> ', () => {
    const comp = (
      <MemoryRouter initialEntries={['/search/query']}>
        <MainWindow />
      </MemoryRouter>
    );
    const wrapper = shallow(comp); // same with render() and just for '/search'
    expect(wrapper.find(SearchBlock)).toHaveLength(1);
```

File: src/client/components/MainWindow/MainWindow.jsx [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
+import {
+  BrowserRouter as Router, Route, Link, Switch, Redirect,
+} from 'react-router-dom';
 import PropTypes from 'prop-types';
 import SearchBlock from '../SearchBlock';
 import SingleMovie from '../SingleMovie';
 import './style.scss';
 
-const MainWindow = ({ showPage }) => (
+const NoMatch = ({ location }) => (
```
Comment (loginov-rocks) [inline src/client/components/MainWindow/MainWindow.jsx]: There should be no more than one component in a file. Please extract it as a separate component.

**Review by loginov-rocks — CHANGES_REQUESTED** [review]
1. There is an error with prop types on `/search`

Screen Shot 2019-04-22 at 1 47 10 PM

2. Search input should contain "star wars" when opening URL with query like this: http://localhost:8080/search/star%20wars - otherwise it's not clear what are the movies displayed on the page.

File: src/client/components/MainWindow/MainWindow.jsx (hunk 1 above)
Comment (vm137) [reply src/client/components/MainWindow/MainWindow.jsx]: fixed.

**Review by loginov-rocks — APPROVED** [review]
LGTM

**Pull request merged**